pyproject.toml

```
@@ -41,18 +28,21 @@ python = "^3.7"
 # required for compatibility
 importlib-metadata = {version = ">=1.7.0", python = "<3.8"}
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
 pre-commit = ">=2.15.0"
-pyrsistent = ">=0.18.0"
+tox = ">=3.0"
+vendoring = {version = ">=1.0", python = "^3.8"}
+
+[tool.poetry.group.test.dependencies]
 pytest = ">=7.1.2"
 pytest-cov = ">=3.0.0"
 pytest-mock = ">=3.5"
```

```suggestion
pytest-mock = ">=3.10"
```

https://github.com/python-poetry/poetry/issues/7929#issuecomment-1547859865

---

Done